Release Notes/Release Notes v2.7.1.md

```diff
@@ -17,3 +17,12 @@ http://mongodb.github.io/mongo-csharp-driver/
 Upgrading
 
 There are no known backwards breaking changes in this release.
+
```

Comment:
These md changes are coming from the v2.7.x branch I presume?

How do we usually handle patch release notes?

Reply:
Yes... these changes (and the missing files below) were in the v2.7.x branch.

I don't think we've had any consistent process for patch release notes, but I would recommend that we make all changes to release notes in master and cherry pick them back to the branch, otherwise master ends up with obsolete or missing release notes.